stdlib/public/Concurrency/Task+TaskExecutor.swift

```diff
@@ -237,6 +237,8 @@ extension Task where Failure == Never {
       addPendingGroupTaskUnconditionally: false,
       isDiscardingTask: false)
 
+//    Builtin.retain(taskExecutor)
```

Comment:
Silly version, keeping existing API and just adding a retain here "works" but we'll instead want to change the parameter to consuming I think